app/main.py

```diff
@@ -1,2 +1,79 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+
+    class Node:
+        def __init__(self, key: Hashable, value: Any) -> None:
+            self.key = key
+            self.value = value
+            self.hash = hash(key)
+
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.size = 0
+        self.load_factor = 2 / 3
```

Comment:
The load factor is set as a fraction (2/3), which will be evaluated as a float (approximately 0.6667). If the intention was to use integer division, you should use `2 // 3` to ensure it evaluates to 0. Consider clarifying the intended behavior here.